.travis.yml

```diff
@@ -39,7 +52,7 @@ before_install:
  #  - Install make
  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
      ./scripts/ci/enable_long_paths.bat;
-     choco install -y python --version=$TRAVIS_PYTHON_VERSION;
```

Comment:
Why change the env var name?

Reply:
To make it more clear it's not actually read to configure macOS or windows python version.